include/sparrow/buffer/dynamic_bitset.hpp

```diff
@@ -561,24 +647,156 @@ namespace sparrow
         zero_unused_bits();
     }
 
+    template <random_access_range B>
+    constexpr void dynamic_bitset_base<B>::clear() noexcept
+    {
+        resize(0);
```

Comment:
This calls buffer's resize which will call the destructor of the storage type. Just making sure I understand correctly: is this always equivalent to just setting every bytes to zero? (maybe we should constrain the storage type to only be a basic type?)

Reply:
I modified the code and is now:
```
m_buffer.clear();
m_size = 0;
m_null_count = 0;
```